Documentation/release-notes/4640.md

```diff
@@ -0,0 +1,5 @@
+#### Application and library build and deployment
+
+  * [GitHub PR 79](https://github.com/xamarin/xamarin-android-tools/commit/f473ff9):
+    May address an `ArgumentException` observed on Windows machine when the
+    Registry contains an invalid path to the Android SDK.
```

Comment:
Thoughts on if it would it be OK to mention the Developer Community item? I was sneaky and added a draft release note idea for that xamarin-android-tools PR in a post-merge [comment there](https://github.com/xamarin/xamarin-android-tools/pull/79#issuecomment-616961208):

```
#### Application and library build and deployment

  * [Developer Community](https://developercommunity.visualstudio.com/content/problem/883179/ilegal-characters-in-path-after-fresh-install.html):
    *System.ArgumentException: Illegal characters in path* could prevent
    successful automatic detection of the Android SDK location during builds in
    some cases if an `AndroidSdkDirectory` registry value was set for
    Xamarin.Android that contained unexpected characters.
```

Reply:
"OK"? Yes?  I was kinda hoping to *avoid* mentioning it, because we're not *entirely* sure it'll fix the problem.  (We *think* it will, but as repro'ing that issue is "weird," we could be wrong.)

Reply:
*In general* I'd say "yes, we should mention devcom issues."  It's *this* case which makes me wary.

Reply:
Ah yeah, that's fair. Works for me. Thanks for the sanity check. Feel free to merge with the current wording.